From e72f364715e7436d21392cde7cfb0c5a3f1a3c1b Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Thu, 30 Nov 2017 16:37:40 +0900
Subject: [PATCH 666/909] arm64: dts: r8a7795-es1-h3ulcb: Disable HS400 of
 SDHI2

commit 254c6cdd0fd68d20a5a161e215b05d774ef6765c from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds the mmc-hs400-1_8v delete-property to SDHI2 and
disable eMMC HS400.

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
index 598b98168559..75a2bc657e30 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
@@ -2,7 +2,7 @@
 /*
  * Device Tree Source for the H3ULCB (R-Car Starter Kit Premier) board
  *
- * Copyright (C) 2016 Renesas Electronics Corp.
+ * Copyright (C) 2016-2017 Renesas Electronics Corp.
  * Copyright (C) 2016 Cogent Embedded, Inc.
  */
 
@@ -35,3 +35,7 @@
 		reg = <0x7 0x00000000 0x0 0x40000000>;
 	};
 };
+
+&sdhi2 {
+	/delete-property/ mmc-hs400-1_8v;
+};
-- 
2.17.1

